Automattic/jetpack
# Payments Button: Remove donation toggle from the block settings

Fixes https://github.com/Automattic/gold/issues/125
Context: p1691608946001779/1691528039.212859-slack-C01B6KEJ5GE

## Proposed changes:

* This patch removes the toggle for "Mark this payment plan as a donation" from the Payment Button block. 

### Other information:


## Jetpack product discussion


n/a

## Does this pull request change what data or activity we track or use?


n/a

## Testing instructions:





1. Switch to this branch.
2. Ensure that your test site is connected to Jetpack/WPCOM.
3. Add a new payment button to a page/post.
4. In the payment button sidebar settings, it should no longer show the option to create a plan as a donation.
![Screen Shot 2023-08-26 at 3 15 05 PM](https://github.com/Automattic/jetpack/assets/8002138/e6a1fe70-9461-445f-a6c0-6512e1fe5737)
